### ISSUE TITLE
Focus indicator not visible during keyboard navigation when using JAWS

### ISSUE THREAD

**Issue by iamrafan** [FreedomScientific/VFO-standards-support#364]
### Summary

1) Open https://codepen.io/iamrafan/pen/RwNdpOK?editable=true
2) Turn on JAWS
3) Navigate to the "New item" button using keyboard and press enter key
4) Observe the focus indicator on the first ```menuitem``` "Email Message" is not visible and upon pressing down arrow, the focus on the second ```menuitem``` "Calender Event" is visible

![JAWS focus bug](https://user-images.githubusercontent.com/22068434/73313846-358b7380-41e1-11ea-81cd-bc82505bf8f0.gif)

### Expected result

The focus indicator on the first ```menuitem``` "Email Message" should be visible.

### Actual result

The focus indicator on the first ```menuitem``` "Email Message" is not visible after activating the "New item" ```menu``` using a keyboard

### Example

https://codepen.io/iamrafan/pen/RwNdpOK?editable=true

or 

the ```button``` under "Contextual Menu button" section in https://developer.microsoft.com/en-us/fabric#/controls/web/button

#### JAWS version and build number
Professional Edition 
Version 2019.1909.28 ILM

 
#### Operating System and version
Windows 10 
Version 1903
OS Build - 18362.476

#### Browser and version:
Chrome - Version 79.0.3945.130

**Comment by JAWS-test** [comment]
I suspect the cause is the switch to application mode due to the ARIA roles for menus. If you use INS+Y to deactivate the virtual cursor and thus no longer switch to application mode, the focus is visible. The problem also occurs with NVDA by the way.
I don't think the problem is serious because screen reader users usually don't need focus visibility.

**Comment by scottaohara** [comment]
With JAWS 2020.2004.66 and Chrome 81 could not reproduce.  JAWS focus (red outline) is visible when moving between menu items.
